cmd/cpr/pending_review.go

```diff
@@ -117,13 +117,13 @@ func PendingReview(token string, dryRun bool) error {
 :red_square: - Closed pull requests
 :purple_square: - Merged pull requests <sup>[1]</sup>
 
-![ovm](https://github.com/prince-chrismc/conan-center-index-pending-review/blob/raw-data/open-versus-merged.gif?raw=true)
+![ovm](https://github.com/` + os.Getenv("GITHUB_REPOSITORY") + `/blob/raw-data/open-versus-merged.gif?raw=true)
```

Comment:
Might be worth messing with https://cli.urfave.org/v2/#values-from-the-environment so it will fail or default when no values are present?